src/engine.rs

```diff
@@ -124,6 +125,8 @@ impl ExitIntoResult for ExitReason {
     }
 }
 
+pub(crate) const RELAYERS_ACCOUNT_PREFIX: &[u8] = b"r";
```

Comment:
I worry a little bit that if we take an ad-hoc approach to key prefixes that we may end up with an accidental collision between keys one day (because the prefix+data of one key happens to match the prefix+data of another key, even if the prefixes and data are different).

I would prefer if we added a new variant to the `KeyPrefix` enum already used in `engine.rs`. If we take a unified approach to key prefixes then a collision will be impossible (because the prefixes will all be the same length and all distinct).

Reply:
Going to have to absolutely agree with that.